source/jormungandr/tests/routing_tests_experimental.py

```diff
@@ -36,14 +36,11 @@
 import jormungandr.scenarios.experimental
 from jormungandr.instance import Instance
 from jormungandr.scenarios.qualifier import min_from_criteria
-
-
-def check_journeys(resp):
-    assert not resp.get('journeys') or sum([1 for j in resp['journeys'] if j['type'] == "best"]) == 1
+from .routing_tests import check_journeys, JourneysWithPtref, Journeys, DirectPath
 
 
 @dataset({"main_routing_test": {'scenario': 'experimental'}})
-class TestJourneysExperimental(AbstractTestFixture):
+class TestJourneysExperimental(Journeys, DirectPath, AbstractTestFixture):
```

Comment:
What I find strange is commenting while on holidays, and not enjoying alex's cakes :)